src/Arcus.Observability.Tests.Integration/Serilog/Sinks/ApplicationInsights/AzureFunctionsDockerTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Azure.ApplicationInsights.Query;
+using Microsoft.Azure.ApplicationInsights.Query.Models;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Arcus.Observability.Tests.Integration.Serilog.Sinks.ApplicationInsights
+{
+    [Collection("Docker")]
+    [Trait("Category", "Docker")]
+    public class AzureFunctionsDockerTests : ApplicationInsightsSinkTests
+    {
+        private static readonly HttpClient HttpClient = new HttpClient();
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AzureFunctionsDockerTests" /> class.
+        /// </summary>
+        public AzureFunctionsDockerTests(ITestOutputHelper outputWriter) 
+            : base(outputWriter)
+        {
+        }
+
+        [Fact]
+        public async Task LogRequest_WithRequestsOperationName_SinksToApplicationInsights()
+        {
+            // Arrange
+            int httpPort = Configuration.GetValue<int>("AzureFunctions:HttpPort");
+            string? requestUri = $"http://localhost:{httpPort}/api/order";
+            Logger.LogInformation("GET -> {URI}", requestUri);
+
+            using (HttpResponseMessage response = await HttpClient.GetAsync(requestUri))
+            {
+                Logger.LogInformation("{StatusCode} <- {URI}", response.StatusCode, requestUri);
+                Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+
+                using (ApplicationInsightsDataClient client = CreateApplicationInsightsClient())
+                {
+                    await RetryAssertUntilTelemetryShouldBeAvailableAsync(async () =>
+                    {
+                        EventsResults<EventsRequestResult> results = await client.Events.GetRequestEventsAsync(ApplicationId, PastHalfHourTimeSpan);
+                        Assert.NotEmpty(results.Value);
+                        AssertX.Any(results.Value, result =>
+                        {
+                            Assert.Contains("order", result.Request.Url);
+                            Assert.Equal(200.ToString(), result.Request.ResultCode);
```

Comment:
Why not just write this as : 
```suggestion
                            Assert.Equal("200", result.Request.ResultCode);
```
?

Reply:
Yeah, is same for me. Integer for the code, as it's a number, and not text. But it's ok your suggestion, too.

Reply:
I was just wondering , as "200" seems more straightforward then `200.ToString` :)